user/installing-dependencies.md

```diff
@@ -23,6 +23,17 @@ before_install:
 >
 > Use the `-y` parameter with apt-get to assume yes as the answer to each apt-get prompt.
 
+
+### Updating APT Packages
+
```

Comment:
In the example before this new content we're running 

```
before_install:
  - sudo apt-get -qq update
  - sudo apt-get install -y libxml2-dev
```

is this to avoid that?